iac-extensions/terraform/src/test/resources/checks/WeakSSLProtocolCheck/azure_mssql_server.tf

```diff
@@ -0,0 +1,23 @@
+resource "azurerm_mssql_server" "mysql_noncompliant_old" {
```

Comment:
I believe names should start with `mssql` and not `mysql` :)

Reply:
Haha nice catch!